department-of-veterans-affairs/va.gov-cms
# VACMS-10579: Adds content validator for links using absolute URLs as relative paths.

## Description

Closes #10579.

## Testing done


## Screenshots


## QA steps


As any user with the ability to edit content:
1. Edit a node, e.g. [VAMC System Health Service Whole health at VA Cincinnati health care](https://cms-pxuj8mm5h1fhqfidhiax67zm0v28bxf4.ci.cms.va.gov/node/24426/edit)
2. Insert a relative link containing a URL into a rich text field.  For instance: `<a href="/https://www.example.org/">Example Link!</a>`.
3. Attempt to save the document.  You should see:
  - [x] An error message at the top of the screen:
![Screen Shot 2023-02-01 at 11 00 38 AM](https://user-images.githubusercontent.com/1318579/216095983-f1e5553f-d72d-42b6-b588-2921a00b159d.png)
  - [ ] validate error message link jumps to the error.

  - [x] An error message at the field:
![Screen Shot 2023-02-01 at 11 01 11 AM](https://user-images.githubusercontent.com/1318579/216096132-45056b54-7b73-4923-8243-a092bf1ad6a4.png)

  - [x] An error message at the bottom of the field:
![Screen Shot 2023-02-01 at 11 00 52 AM](https://user-images.githubusercontent.com/1318579/216096044-8e025386-92d1-45a0-8185-e5760d71fb34.png)

4. Edit a media image item, e.g. [August2022.jpg](https://cms-pxuj8mm5h1fhqfidhiax67zm0v28bxf4.ci.cms.va.gov/media/image/16769)

5.  Insert a relative link containing a URL into a plain text field. For instance, `/https://www.example.org/`.

6. Attempt to save the media image item.  You should see:
  - [x] An error message at the top of the screen:
![Screen Shot 2023-02-01 at 11 02 47 AM](https://user-images.githubusercontent.com/1318579/216096625-f5099f31-06ec-46c8-a507-1f34a2d28896.png)
  - [ ] validate error message link jumps to the error.

  - [x] An error message at the field:
![Screen Shot 2023-02-01 at 11 02 51 AM](https://user-images.githubusercontent.com/1318579/216096611-e403b3f3-5f61-446a-b691-1ecf4f1e4d4d.png)

  - [x] An error message at the bottom of the field:
![Screen Shot 2023-02-01 at 11 02 56 AM](https://user-images.githubusercontent.com/1318579/216096598-f4fe79d8-b0f1-413a-9e85-98e4e0be7acf.png)

### Definition of Done


### Select Team for PR review
